libshaderc/src/shaderc_cpp_test.cc

```diff
@@ -79,7 +80,10 @@ class CppInterface : public testing::Test {
   // message.
   std::string CompilationErrors(const std::string& shader,
```

Comment:
Change the function name to CompilationWarnings() or I can change it and add another CompilationErrors() later?